src/View/Helper/ToolbarHelper.php

```diff
@@ -77,6 +88,9 @@ public function makeNeatArray($values, $openDepth = 0, $currentDepth = 0, $doubl
         if (empty($values)) {
             $values[] = '(empty)';
         }
+        if ($this->sort && is_array($values)) {
+            ksort($values);
+        }
```

Comment:
I don't know if more checkboxes is a great experience. The one checkbox should be fine for now.